pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller_test.go

```diff
@@ -31,12 +31,12 @@ import (
 
 	"github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
 	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
-	"github.com/onsi/gomega"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	//"github.com/onsi/gomega"
```

Comment:
Same thing here, anywhere we are doing mass commenting out I'd suggest just removing those lines.